lk2nd/device/dts/msm8952/msm8937-mtp.dts

```diff
@@ -12,9 +12,22 @@
 	cedric {
 		model = "Motorola Moto G5 (cedric)";
 		compatible = "motorola,cedric";
-		lk2nd,match-device = "cedric";
```

Comment:
I'm confused. I'd expect device model matching be way more reliable than panel matching (which should only be used for the last resort, when nothing else can be used.) Why are you switching the matching here?

Reply:
Oops. I suggested to use either of them instead of both in earlier comments.